src/endpoints/Billing/types.ts

```diff
@@ -1,12 +1,28 @@
+export enum SignupPlan {
+    STARTER = 'starter',
+    CORE = 'core',
+    PREMIUM = 'premium',
+}
+
+export enum SignupCurrency {
+    EUR = 'eur',
+    USD = 'usd',
+}
+
+export enum SignupBillingCycle {
+    YEAR = 'year',
+    MONTH = 'month',
+}
+
 export interface SignupRequest {
     email: string;
     company_name: string;
     first_name: string;
     last_name: string;
     quantity: number;
-    plan?: 'core';
-    currency?: 'eur';
-    billing_cycle?: 'month';
+    plan?: SignupPlan;
+    currency?: SignupCurrency;
+    billing_cycle?: SignupBillingCycle;
```

Comment:
Just to be sure - these are not required in the API, right? I saw them being required in the first commits 🤔

Reply:
well, technically they are mandatory
<img width="692" alt="image" src="https://user-images.githubusercontent.com/3620639/204787714-225616a9-a8a0-462c-b861-24fe4b7f51c6.png">

i thought that should be changed in future?